<!--StartFragment-->

Structural Insights into the IGF1R:  A Comparative Analysis of Homology Modelling and Alphafold

Authors: Oleh Precious (@KelHills), Bezaleel Akinbami (@B3z), Rukayat Omotosho-Sanni(@Rxruqayyah), Itrat Zehrh (@Zehrh \_iz), Cynthia Eriobu (@Cynthia1000)


## **INTRODUCTION** {#h.cjenfkg3wiwy}

Insulin-like Growth Factor (IGF) is a polypeptide hormone that shares similar molecular characteristics with proinsulin \[1]. IGF consists of two forms, IGF-1 and IGF-2, each with its own specific receptors (IGF-1R and IGF-2R) and binding proteins (IGFBP1–IGFBP6), which regulate their bioavailability and activity \[2]. The IGF-1 receptor (IGF1R) consists of several key domains that present significant potential as therapeutic targets for drug development. These domains include the leucine-rich repeat 1 (L1) domain, the cysteine-rich domain (CRD), fibronectin type III domains (FNIII-1 and FNIII-2), and the tyrosine kinase domain (TKD). Each of these domains plays a critical role in the receptor’s function, contributing to IGF-1 binding, receptor dimerization, stability, ligand interaction, catalytic activity, and ATP binding.The primary IGF1-binding site observed in our IGF1R-IGF1 cryo-EM structure consists of the L1 domain and α-CT′ and is nearly identical to that in the crystal structure of the IGF1R ectodomain bound to IGF1. At this secondary IGF1-binding sub-site, Y487 and R488 of IGF1R pack closely against residues T4, C6, and E9 of IGF1. \[3]. 


## **METHODOLOGY** {#h.9vcw9u4czp8l}

- IGFR1 in Apo, Agonist, and Antagonist forms was retrieved from the PDB (Protein Data Bank) with the ID 1P4O for the Apo (unactivated), 6PYH for the agonist, and 3LVP for the Antagonist forms. Their FASTA sequence was collected. 

- Homology modeling of each protein variant was performed using SWISS-MODEL, based on the provided sequence.

- The AlphaFold model, which had already been predicted, was retrieved from the AlphaFold database.

- Structural visualization was conducted in PyMOL, and the modeled structures were compared to the crystal structures available in the PDB using alignment and RMSD measurements.


## **RESULT** {#h.9swqol5bj4k9}

****![](https://lh7-rt.googleusercontent.com/docsz/AD_4nXfIi40Kxw86Pyi87IdqyO6tqas57C9xWwTckKSEd3gsSEcmnqzojx7oiqqvRjihS_eRoJuQ9Y82ZkNJOxoRpzs5ONGad3utuP9Sg9sxFn0IVqscfn197daVkjkAYTturDKpln80Sm8oEhS2DV2mqf200ME?key=Fa1xiTQHd0grh7ITfhJq9w)****

****![](https://lh7-rt.googleusercontent.com/docsz/AD_4nXc2ZzgiZHbfJguuTXi8bjwwezt-ApZKRkFJlUqpVZEA_-tEG6MB8mvo1qvY9GN2p_-ON7zxg24E7E1McOoovk2VzoQFcT3D7ke1aEMe-7HZS-jbp6U0FOo7YBQ_ffTWMAeekyBjDC7cS3s-XsVND8DKw5RS?key=Fa1xiTQHd0grh7ITfhJq9w)****

**Figure 1: Structural image of Apo IGF1R (1P4O)**   

****![](https://lh7-rt.googleusercontent.com/docsz/AD_4nXdwYpeznqKUArXqNb6ZYmZw7-bS3KeSxfGeCl0mm0nDICb7FpH_rGGyl8YvDOkkOctn0Mi5MZRA1J1IbQAo565hSeHCYSO7JTZt3vH3jnLgqJshCDvYrwB0f9r9vEOc_GILxeqSuKD7a6dD2Su9m4D4JE0?key=Fa1xiTQHd0grh7ITfhJq9w)****

**Figure 2: Structural Image of Agonist IGF1R (6PYH)**

****![](https://lh7-rt.googleusercontent.com/docsz/AD_4nXcNFO2xquNuROGyzxmGajFBTSao2vVrVdekI2Z92LoCB61t3h5XL7SqsZbTS6xTmoMC5bRou6W41T5sE54T_bUNLxQn8FygngpyGP94JX3kusUh0qqpBPLjXcCsqEAt9xWRfvID2qOH4H0rQshf0E7i-9SO?key=Fa1xiTQHd0grh7ITfhJq9w)****

**Figure 3:Structural Image of Antagonist IGF1R (3LVP)**

****![](https://lh7-rt.googleusercontent.com/docsz/AD_4nXcaMWexlA6V52RxjQ4RvsJ_B9ZYER7zHCn8_-gPVrUAaBB1Yf-9ighimXjfwtB09q6lDBapJPETRVoX-6Y2lxfDJlej0nmA53oNwIFSVQ3wmlTo3FazrIohYsKdErxzAIqYZvwbii_2kxtbTXTBhnvDdzwK?key=Fa1xiTQHd0grh7ITfhJq9w)****

**Figure 4: Structural image of IGF1R, modeled with SWISS**

****![](https://lh7-rt.googleusercontent.com/docsz/AD_4nXcEumEexYCMbKe3Vi-jcYz4vVhbnu6U2UpT_mZEpD7b3hTNFouGHl8LxhCnUIotID5qMRkPmPIkV1cUgIkypRTgfmOeOQ2bQ9K3u1lFWLgGa00Tnt0yy3w-TRqCoGz0cOCwxHjO0Mi8uegkPXPfdxM5YTE?key=Fa1xiTQHd0grh7ITfhJq9w)****

**Figure 5: Structural image of IGF1R, modeled with AlphaFold**

\



## **DISCUSSION**  {#h.db6b04d898ci}

The analysis indicates that the protein structure predicted by AlphaFold adopts an apo conformation, with an RMSD value of 1.014, while the structure generated by SWISS-MODEL exhibits an agonist conformation, with a significantly higher RMSD value of 18.810. With reference to the RMSD values, the AlphaFold model had lower values and could be said that it is more accurate and useful for modeling the IGF1R protein.


## **CONCLUSION** {#h.av93wqejcp7a}

The IGF1R protein has been modeled using the AlphaFold and Swiss modeling techniques. The accuracy of these techniques has been established for IGF1R protein with AlphaFold taking precedence over SWISS modeling.

\
\
\


 ****

        ****

\
\



## **REFERENCES** {#h.adhfepxhwytf}

\[1] ********J. E. Puche and I. Castilla-Cortázar, ‘Human conditions of insulin-like growth factor-I (IGF-I) deficiency’, J. Transl. Med., vol. 10, no. 1, p. 224, Nov. 2012, doi: 10.1186/1479-5876-10-224.

\[2] J. Bailes and M. Soloviev, ‘Insulin-Like Growth Factor-1 (IGF-1) and Its Monitoring in Medical Diagnostic and in Sports’, Biomolecules, vol. 11, no. 2, p. 217, Feb. 2021, doi: 10.3390/biom11020217.

\[3] J. Li, E. Choi, H. Yu, and X. Bai, ‘Structural basis of the activation of type 1 insulin-like growth factor receptor’, Nat. Commun., vol. 10, p. 4567, Oct. 2019, doi: 10.1038/s41467-019-12564-0.

\
\
\
\


<!--EndFragment-->
